### PR TITLE
feature/metrics

### PR DESCRIPTION
**Pull request by kvahed**

### CODE REVIEW

File: arangod/RestServer/dynarray.h [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#include <iterator>
```
Comment (neunhoef) [inline arangod/RestServer/dynarray.h]: Header with copyright missing

File: arangod/RestHandler/RestMetricsHandler.cpp [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+#include "Agency/AgencyFeature.h"
+#include "Agency/Agent.h"
+#include "ApplicationFeatures/ApplicationServer.h"
+#include "Cluster/ServerState.h"
+#include "GeneralServer/ServerSecurityFeature.h"
+#include "Rest/Version.h"
+#include "RestServer/ServerFeature.h"
+
+#if defined(TRI_HAVE_POSIX_THREADS)
+#include <unistd.h>
```
[the remark this answers is not in the document]
Comment (kvahed) [reply arangod/RestHandler/RestMetricsHandler.cpp]: forgot no longer needed.

File: arangod/RestHandler/RestMetricsHandler.h [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+#include "RestServer/MetricsFeature.h"
+
+namespace arangodb {
+class RestMetricsHandler : public arangodb::RestBaseHandler {
+ public:
+  RestMetricsHandler(application_features::ApplicationServer&, GeneralRequest*,
+                     GeneralResponse*);
+
+  char const* name() const override final { return "RestMetricsHandler"; }
+  RequestLane lane() const override final { return RequestLane::CLIENT_FAST; }
```
[the remark this answers is not in the document]
Comment (kvahed) [reply arangod/RestHandler/RestMetricsHandler.h]: Not much thought went into this. Lowering.

File: arangod/RestServer/dynarray.h (hunk 1 above)
Comment (kvahed) [reply arangod/RestServer/dynarray.h]: done

**Comment by kvahed** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/7294/

**Pull request merged**